src/frontend-admin/src/components/reusables/Card/Card.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styles from './Card.module.css'
+import {Title} from '../Title/Title' 
+
+interface ICardProps {
+    title: any
+    children: any //any so that dropdown can be used
+}
```

Comment:
One other prop you may consider adding here (and really any widely reusable component) is an optional `className`. That way should we ever need to override any stylings when using a card we can easily do so. You can then use your classnames package to concatenate this optional prop with `styles.cardMain`.